src/components/BAccordion/BAccordion.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <section
+    :id="id"
+    role="heading"
+  >
+    <button
+      :id="`${_uid}-heading`"
+      class="header"
+      :aria-expanded="expanded"
```

Comment:
Somehow instead of updating the aria-expanded value to false, it just removes it. Is there a way to not let it do that?